src/Command/Ssh/SshKeyCreateCommand.php

```diff
@@ -128,18 +128,21 @@ protected function validateFilename($filename) {
    * @throws \Exception
    */
   protected function determinePassword(InputInterface $input, OutputInterface $output): string {
+    $password = "";
     if ($input->getOption('password')) {
       $password = $input->getOption('password');
       $this->validatePassword($password);
     }
     else {
-      $question = new Question('Enter a password for your SSH key');
```

Comment:
We still want to prompt people for a password when they are doing this on their local machine.

Reply:
Now it will prompt for the password if user tries to create the SSH key for local machine.